codespell_lib/data/dictionary_code.txt

```diff
@@ -2,6 +2,15 @@ amin->main
 cas->case, cast,
 clas->class
 cmo->com
+daemon->demon
```

Comment:
This isn't a typo ever:
https://en.wikipedia.org/wiki/Daemon_(classical_mythology)
https://en.wikipedia.org/wiki/Daemon_(computing)